.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1 +0,0 @@
-blank_issues_enabled: false
```

Comment:
This was added because users would ignore the template, rather than pick the one that they should have used

Reply:
Yes I know but sometimes the template doesn't fit and in that case it's annoying. It's also a rather small link at the bottom under the templates.

Additionally, even with the templates we see people ignoring them by just putting random stuff in the boxes and then editing them out.